source/globalCommands.py

```diff
@@ -4382,6 +4386,18 @@ def script_cycleParagraphStyle(self, gesture: "inputCore.InputGesture") -> None:
 		config.conf["documentNavigation"]["paragraphStyle"] = newFlag.name
 		ui.message(newFlag.displayString)
 
+	@script(
+		description=_(
+			# Translators: Describes a command.
+			"Cycles through sound split modes",
+		),
+		category=SCRCAT_AUDIOSTREAMING,
+		gesture="kb:NVDA+alt+s",
+	)
+	def script_cycleSoundSplit(self, gesture: "inputCore.InputGesture") -> None:
+		import audio

```

Comment:
Can't this be imported at the module level? If there is a circular import making this difficult please add a comment stating that this is the case. In terms of performance this should not be noticeable, as `audio` is unconditionally imported in `core.main`.